### DISCUSSION TITLE
Best practices to handle a book with large number of SVG files

### DISCUSSION THREAD

**Discussion by rgaiacs** [quarto-dev/quarto-cli#1778] [Q&A] [answered]
Hi,

I'm curious to know what is the recommended practice to handle a book with large number (> 100) of SVG files and that need to be converted to HTML, DOCX, and PDF.

My previous workflow with bookdown was to use `make` to convert the SVG with Inkscape to PNG and PDF. And I would include the PNG files in my R Markdown document. I only nee to generate the PDF at the end of the project and I would use `sed` to replace `.png` with `.pdf`.

Thanks for the suggestions.

**Comment by jjallaire** [comment, marked as answer]
You can specify images without an extension and in that case they will use the `default-image-extension` for the format. See https://quarto.org/docs/authoring/figures.html#multiformat-figures
